packages/relay/src/lib/eth.ts

```diff
@@ -285,13 +289,12 @@ export class EthImpl implements Eth {
     return !CommonService.blockTagIsLatestOrPendingStrict(tag) && !CommonService.isDevMode;
   }
 
-  private initEthExecutionCounter(register: Registry): Counter {
-    const metricCounterName = 'rpc_relay_eth_executions';
+  private initCounter(metricCounterName: string, labelNames: string[], register: Registry): Counter {
```

Comment:
qq: why did we want to rename the method?

Reply:
firstly, I implemented a second counter and reworked this method so I can reuse it and not have repetitive code. Afterwords we decided to use one counter with different labels. I left this change, so we can use it if we decide to refactor the metrics service

Reply:
Nice! sounds good to me! Thanks for the explanation!